pkg/webui/components/profile-dropdown/index.js

```diff
@@ -26,45 +25,22 @@ import PropTypes from '@ttn-lw/lib/prop-types'
 import styles from './profile-dropdown.styl'
 
 const ProfileDropdown = props => {
-  const [expanded, setExpanded] = useState(false)
-  const node = useRef(null)
   const { brandLogo, className, children, profilePicture, ...rest } = props
 
-  const handleClickOutside = useCallback(e => {
```

Comment:
There was quite some logic duplication in here with regards to show/hiding the dropdown. I've refactored this to use the existing logic of the dropdown component.